bin/install_npm_packages

```diff
@@ -2,13 +2,13 @@
 
 # Ensure package.json is present
 if [ ! -f package.json ]; then
-    echo "No valid package.json file found"
+    echo "No package.json found!"
```

Comment:
Me being pedantic:

- No validation made on `package.json`
- We know that `package.json` is a file